web_widget_dropdown_dynamic/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+Dynamic dropdown widget that supports resolving options from backend of:
+
+ * `fields.Char`
+ * `fields.Integer`
+ * `fields.Selection`
```

Comment:
I doubt this one is supported when you save a value that is not on the original selection.

Reply:
You are correct, saving a value that does not exist in original `selection` will give an error, which is by design. This field can be used only to _filter_ selection, not to extend. For fully-dynamic fields, use `Char` or `Integer`

Reply:
You can put a note about that.